apps/smart-contracts/core/test/fixtures/HookFixture.ts

```diff
@@ -24,9 +24,9 @@ export async function managerWithdrawHookFixture(): Promise<ManagerWithdrawHook>
   return (await managerWithdrawHook.deploy()) as ManagerWithdrawHook
 }
 
-export async function feeRebateHookFixture(): Promise<FeeRebateHook> {
-  const feeRebateHook = await ethers.getContractFactory('FeeRebateHook')
-  return (await feeRebateHook.deploy()) as FeeRebateHook
+export async function tokenSenderCallerFixture(): Promise<TokenSenderCaller> {
+  const tokenSenderCallerHook = await ethers.getContractFactory('TokenSenderCaller')
+  return (await tokenSenderCallerHook.deploy()) as TokenSenderCaller
 }
```

Comment:
```suggestion
export async function tokenSenderCallerFixture(): Promise<TokenSenderCaller> {
  const factory = await ethers.getContractFactory('TokenSenderCaller')
  return (await factory.deploy()) as TokenSenderCaller
}
```
As per other PRs, I know this is a departure of most of the others, but trying to slowly change over since no point in giving it a different name when it only exists within this context.